From 484623eda09d8e8b9857f5279ebb891a423588dc Mon Sep 17 00:00:00 2001
From: Kylo Ginsberg <kylo@puppet.com>
Date: Sun, 18 Sep 2016 06:42:46 -0700
Subject: [PATCH] (maint) Remove unused: requre 'xmlrpc/client'

In 152299cc, as part of PUP-6120, use of the xmlrpc library
was removed from the code, but the require itself was not
removed. This commit simply removes the vestigial require.

Note that this change is required to support ruby 2.4,
which no longer includes the xmlrpc library.
---
 lib/puppet/provider/package/pip.rb | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lib/puppet/provider/package/pip.rb b/lib/puppet/provider/package/pip.rb
index 9300b7b..aad447b 100644
--- a/lib/puppet/provider/package/pip.rb
+++ b/lib/puppet/provider/package/pip.rb
@@ -2,7 +2,6 @@
 # <http://pip.pypa.io/>
 
 require 'puppet/provider/package'
-require 'xmlrpc/client'
 require 'puppet/util/http_proxy'
 
 Puppet::Type.type(:package).provide :pip,
-- 
2.7.4

